src/components/ui/ButtonsGroup/ButtonsGroup.test.tsx

```diff
@@ -0,0 +1,51 @@
+import { render, screen } from '@testing-library/react';
+import ButtonsGroup from './index';
+
+describe('ButtonsGroup', () => {
+  it('renders with left text and right text', () => {
+    render(
+      <ButtonsGroup
+        leftText={'foo'}
```

Comment:
Let's format it and remove curly braces

Reply:
Prettier is not stripping off these braces automatically. I advise we don't change this now or change prettier config